lib/assessment.rb

```diff
@@ -5,7 +5,9 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.reduce(0) do |memo, item|
+      item.length > memo ? item.length : memo
+    end
```

Comment:
This approach is _almost_ right - the problem is that it will return the _length_ of the longest element, rather than the element itself. Can you think of how you might change your code to get that result?